### ISSUE TITLE
WCAG 2.2 Understanding Success Criterion 2.5.8: Target Size (Minimum)

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#2740]
Intent section:
In the paragraph right above the image of the examples, the last two sentences have "the the", should be "the".

Examples section:
The text that exists now:

> A row of buttons, each of which has a horizontal width of more than 24 CSS pixels, but a height of only 20 CSS pixels. Since there is sufficient spacing above and below the row of buttons, the Success Criterion passes.

Suggested text to clarify what the passing situation is before saying it passes.
>A row of buttons, each of which has a horizontal width of more than 24 CSS pixels, a height of only 20 CSS pixels, and vertical padding of 2 CSS pixels above and 2 CSS pixels below the row of buttons. Since there is sufficient spacing above and below the row of buttons, the Success Criterion passes.

**Comment by alastc** [comment]
Merged #2743 so closing.
